frame/babe/src/lib.rs

```diff
@@ -835,3 +859,46 @@ fn compute_randomness(
 
 	sp_io::hashing::blake2_256(&s)
 }
+
+pub mod migrations {
+	use super::*;
+	use frame_support::pallet_prelude::{ValueQuery, StorageValue};
+
+	pub trait HasPalletPrefix: Config {
```

Comment:
Weird naming, no docs.  

```suggestion
	/// Something that can return the storage prefix of the `Babe` pallet.
	pub trait PalletPrefix: Config {
```